addon/mixins/pikaday.js

```diff
@@ -136,14 +137,19 @@ export default Mixin.create({
   },
 
   setMinDate: function() {
-    const { pikaday, minDate, value } = getProperties(this, [
+    let { pikaday, minDate, value } = getProperties(this, [
       'pikaday',
       'minDate',
       'value'
     ]);
 
     if (minDate) {
-      const _minDate = new Date(minDate.getTime());
+      try {
+        minDate = minDate.getTime();
+      } catch (e) {
```

Comment:
Is there a specific error you're looking to catch here? I'm not a fan of this blanket approach of catching _any_ error, ignoring it, and doing a different date conversion.

If there is something specific we're relying on an exception for here, can we test that case in some other way than calling `getTime` and catching a potentially resulting error?

I don't want to use exception handling for control flow.